frontend/device/generic/device.lua

```diff
@@ -25,6 +25,7 @@ local Device = {
     hasFrontlight = no,
     needsTouchScreenProbe = no,
     hasClipboard = yes, -- generic internal clipboard on all devices
+    hasEinkScreen = no,
```

Comment:
Interesting default, btw.

Reply:
Is something like generic: yes; android fallback: no a viable alternative?

Reply:
yes, android.isEink() will return false until the device is added to a list of known supported devices. So yes for generic, android.isEink() for android, no for generic sdl and yes for the emulator?

Reply:
Sounds good to me.